nautobot/extras/management/__init__.py

```diff
@@ -171,38 +168,26 @@ def create_custom_statuses(
 
         for choice_kwargs in choices:
```

Comment:
May need to revert some or all of the changes to this file in order to get initial status creation (from a fresh DB) to work as intended.